spec/index.bs

```diff
@@ -15811,7 +15813,10 @@ This feature adds the following [=optional API surfaces=]:
 <span id=dom-gpufeaturename-texture-compression-bc></span>
 </h3>
 
-Allows for explicit creation of textures of BC compressed formats. Supports both 2D and 3D textures.
+Allows for explicit creation of textures of BC compressed formats. Only supports 2D textures.
+
+Note: This feature implies the availability of
+the {{GPUFeatureName/"texture-compression-bc-sliced-3d"}} feature.
```

Comment:
nit for clarity on two things:
- we're talking about what is available on adapters (not devices)
- what does "implies" mean

```suggestion
Note: Adapters which support either {{GPUFeatureName/"texture-compression-bc"}}
or {{GPUFeatureName/"texture-compression-bc-sliced-3d"}}
always support both, even though they are separate features.
They still must be enabled separately.
```